app/models/job_invocation.rb

```diff
@@ -75,6 +75,8 @@ class JobInvocation < ApplicationRecord
 
   default_scope -> { order('job_invocations.id DESC') }
 
+  scope :latest_jobs, -> { unscoped.joins(:tasks).order('foreman_tasks_tasks.started_at DESC').authorized(:view_job_invocations).limit(5) }
```

Comment:
This doesn't seem to work 
![image](https://user-images.githubusercontent.com/7326770/127174438-f85f8032-cfd6-497f-a3d1-4a327a27cfef.png)

It is possible that NULLs go first and are shown in some implicit order and then the actual shown value comes from `foreman_tasks_task.start_at`. Maybe using that would be a better option?

Reply:
With these suggestions it creates this query:
```sql
SELECT "job_invocations".* FROM "job_invocations" INNER JOIN "foreman_tasks_tasks" ON "foreman_tasks_tasks"."id" = "job_invocations"."task_id" ORDER BY foreman_tasks_task.start_at DESC LIMIT $1  [["LIMIT", 5]]
```

but I think it should join with `foreman_tasks_task`, because this throws error

Reply:
Oh, sorry, my bad. That was a typo. The table you're using is correct, but a different field (`start_at` instead of `started_at`) would most likely give better results.

Reply:
Oh okay, now it should be good :+1: